iac-extensions/docker/src/main/java/org/sonar/iac/docker/tree/impl/HealthCheckInstructionImpl.java

```diff
@@ -54,10 +69,17 @@ public List<Flag> options() {
   }
 
   @Override
-  public Instruction instruction() {
+  @CheckForNull
+  public CmdInstruction instruction() {
     return instruction;
   }
```

Comment:
What about renaming this one to `cmdInstruction` or something more precise ? Up to you, I don't have a strong opinion on this.